src/modules/evbc/simulator-sample-data/sample_config_list.ts

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2020 - 2024 Pionix GmbH and Contributors to EVerest
 
-import { EverestConfigList } from ".";
+import {EverestConfigList} from "../index";
```

Comment:
I guess we already have an autolint running so these changes can be ignored from a reviewer's perspective?

Reply:
Will implement soon